src/metadata.rs

```diff
@@ -311,6 +311,34 @@ impl<'de> Deserialize<'de> for PlatformWithVirtualPackages {
     }
 }
 
+/// A newtype wrapper around a boolean indicating whether debug output is enabled
+#[derive(Debug, Clone, Copy, Default, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Debug(bool);
+
+impl Debug {
+    /// Create a new Debug instance
+    pub fn new(debug: bool) -> Self {
+        Self(debug)
+    }
+
+    /// Returns true if debug output is enabled
+    pub fn is_enabled(&self) -> bool {
+        self.0
+    }
+}
+
+impl From<bool> for Debug {
```

Comment:
Can we remove these methods? Both `From<bool>` and `From<Debug>`? Then we have even more type safety! :)